src/chains/ethereum/ethereum/tests/api/eth/createAccessList.test.ts

```diff
@@ -0,0 +1,418 @@
+import assert from "assert";
+import { EthereumProvider } from "../../../src/provider";
+import getProvider from "../../helpers/getProvider";
+import compile, { CompileOutput } from "../../helpers/compile";
+import { join } from "path";
+import { Quantity } from "@ganache/utils";
+import { Ethereum } from "../../../src/api-types";
+import { AccessList } from "@ganache/ethereum-transaction/src/access-lists";
+
+const encodeValue = (val: number | string) => {
+  return Quantity.from(val).toBuffer().toString("hex").padStart(64, "0");
```

Comment:
```suggestion
  return Quantity.toBuffer(val).toString("hex").padStart(64, "0");
```

If you don't require the special cases for `Quantity` type (ie `0x0` and nullable), you could use the ByteLength parameter of `Data`

```suggestion
  return Data.toString(val, 32).slice(2);
```
Will return just the hex component, padded / truncated to exactly 64 chars